src/components/patch-assignment-form/patch-assignment-form.tsx

```diff
@@ -46,20 +46,23 @@ export const PatchAssignmentForm = ({ setShowSuccess, setRequestError }: Props)
     .sort((a, b) => (a.name > b.name ? 1 : -1))
     .sort((a, b) => (a.patchType !== "area" && b.patchType === "area" ? 1 : -1));
 
-  function displaySubmissionStatus(success: boolean, patch: Patch, error?: Error) {
-    if (success) {
-      const matchingPatch = patchesAndAreas.find(
-        (patchOrArea) => patchOrArea.id === patch.id,
-      );
-      if (!matchingPatch) return;
-      matchingPatch.responsibleEntities[0] = patch.responsibleEntities[0];
-      setPatchesAndAreas([...patchesAndAreas]);
-      setShowSuccess(true);
-      setReassigningPatch(null);
-    } else {
-      setRequestError(error?.message || "An error occurred");
-    }
-  }
+  const displaySubmissionStatus = useCallback(
```

Comment:
What do we get for changing this into `useCallback`?

Reply:
There  was linting error,s so I had to change the code